frontend/src/components/widget/Lookup/RawLookup.js

```diff
@@ -457,7 +461,7 @@ export class RawLookup extends Component {
         { isInputEmpty: false, loading: true, query: inputValue },
         () => {
           const query = this.state.query;
-          if (query.length >= this.minQueryLength) {
+          if (query.length >= this.minQueryLength || allowEmpty) {
```

Comment:
what `allowEmpty` does and what's the logic here?

Reply:
to ensure that `this.autocompleteSearchDebounced()` is called even when the input is empty.